docs/src/userguide/language_basics.rst

```diff
@@ -556,6 +556,8 @@ possibilities.
 +----------------------------+--------------------+------------------+
 | char*                      | str/bytes          | str/bytes [#]_   |
 +----------------------------+--------------------+------------------+
+| C array (other than char*) | iterable           | list [#2]_       |
```

Comment:
`other than char*` seems misleading here, since it's referring to a pointer and not an array specifically. -> `char[]`
Should this also mention `signed/unsigned char[]`, just for completeness?